src/test/steps/migrate_app_js.test.ts

```diff
@@ -308,6 +308,17 @@ describe("migrate app js", () => {
                      }`)
                   );
                 });
+                it("should correctly chain together invoke calls", async () => {
+                  writeFixtureSrc(`foo: function() {
+                    this.${api}("sharedWith").hide();
```

Comment:
just a note for you to double check what kind of apis and arguments can be chained, what about multiple arguments